README.md

```diff
@@ -377,6 +377,9 @@ These can be useful for osint and social engineering.
 - [Sogou](https://pic.sogou.com/) - Chinese technology company that offers a search engine
 - [Jimpl](https://jimpl.com/) - Online photo metadata and EXIF data viewer
 - [Same Energy](https://same.energy/) - Find beautiful images
+- [Exid data](http://exifdata.com/)- look at your favorite images!
```

Comment:
The description should be `online applicatation that lets you take a deeper look at your favorite images`